src/main/java/frc/robot/subsystems/funnel/Funnel.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.subsystems.funnel;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/**
+ * Name new TalonSRX and input his data from Ports.
+ */
+public class Funnel extends SubsystemBase {
+
+    public final TalonSRX motor = new TalonSRX(Ports.Motor.MOTOR);
+
+    /**
+     * set motor power [%]
+     * @param power the speed times the torque.
+     */
+    private void setMotorPower(double power) {
+        motor.set(ControlMode.PercentOutput, power);
+    }
+
+    /**
+     * motor boolean inverted input data from Ports.
+     */
+
+    private void setMotorInverted() {
+        motor.setInverted(Ports.Motor.IS_MOTOR_INVERTED);
+
+    }
```

Comment:
Create a constructor for Funnel and set the inversion there. Separating those things might be useful but only when you have a lot of inversions and you want to keep your constructor clean, in any case you'd still need to call these functions in the constructor.